tests.mk

```diff
@@ -49,6 +49,27 @@ test_p2p:
 	#
 	# mkdir -p test/p2p/logs && docker cp rsyslog:/var/log test/p2p/logs
 
+test_p2p_ipv6:
+	# IPv6 tests require Docker daemon with IPv6 enabled, e.g. in daemon.json:
+	#
+	# {
+	#	"ipv6": true,
+	#   "fixed-cidr-v6": "2001:db8:1::/64"
+	# }
+	#
+	# Docker for Mac can set this via Preferences -> Docker Engine.
+	docker rm -f rsyslog || true
+	rm -rf test/logs || true
+	mkdir test/logs
+	cd test/
```

Comment:
won't it be better to chain the following three commands?

```
cd test/ &&
  docker ... &&
  cd ..
```

Reply:
I removed the cd'ing altogether. This was just copied from `test_p2p`.